stdlib/LinearAlgebra/src/bunchkaufman.jl

```diff
@@ -4,6 +4,48 @@
 ## LD for BunchKaufman, UL for CholeskyDense, LU for LUDense and
 ## define size methods for Factorization types using it.
 
+"""
+    BunchKaufman <: Factorization
+
+Matrix factorization type of the Bunch-Kaufman factorization of a symmetric or
+Hermitian matrix `A` as ``P'*U*D*U'*P`` or ``P'*L*D*L'*P``, depending on
```

Comment:
It's a bit more general since it can also factorize complex symmetric matrices in which case it should be `transpose` and not `'`.

Reply:
Since this is in math mode, can I use ``P^\\top`` or ``P^T``. Or should I turn it simply into code mode and use `transpose`? That mixture of code and math looks a bit weird anyway. Normally, we wouldn't write the `*` in math mode, would we?

Reply:
I missed to adopt the corresponding parts from the `bunchkaufman` docstring, because I initially felt that this relates more to the factorization function, and less to the factorization type (see above). But I agree that this should be mentioned, and I have added it. I also turned the math type into code type, and did the same to the factorization function docstring. There, I also changed the wording slightly, to reflect the fact that the matrix should be symmetric/Hermitian, not necessarily of `Symmetric` or `Hermitian` type. I hope that with the added doctest example, the issue with the stored triangle and the wrapping by the special types becomes clearer.